### DISCUSSION TITLE
Disable codeium automatically

### DISCUSSION THREAD

**Discussion by madflow** [LazyVim/LazyVim#2757] [Q&A] [answered]
- I would like to disable "Codeium" automatically.
- Maybe when there is a certain file in a root directory? `.no-codium` or something.
- Does anyone have a lua codesnippet for this?
- Maybe more in general: How to disable a plugin conditionally?

**Comment by dpetka2001** [comment, marked as answer]
You can use `lazy.nvim`'s `cond` property (see [docs](https://github.com/folke/lazy.nvim#-plugin-spec)). I do something similar with the git plugins that I use (see [here](https://github.com/dpetka2001/dotfiles/blob/main/dot_config/nvim/lua/plugins/git.lua) for examples). 

I use `codeium.vim` and not `codeium.nvim`, because I found that didn't want just another source in `nvim-cmp` to pollute the completion-menu and make it even more sluggish (at least on my old laptop, might not be the case for everyone). In my config I customized it so that it only shows completions when the `nvim-cmp` completion menu is not open as you can see [here](https://github.com/dpetka2001/dotfiles/blob/5acae05aecd4912b9275bf190390814625338b5a/dot_config/nvim/lua/plugins/coding.lua#L66-L74).

Hope these examples serve as some inspiration as to what you want to achieve.

**Reply by madflow** [reply]
💯 👍
